lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -565,7 +568,8 @@ if (require.main === module) {
   const coreStrings = collectAllStringsInDir(path.join(LH_ROOT, 'lighthouse-core'));
   console.log('Collected from LH core!');
 
-  const stackPackStrings = collectAllStringsInDir(path.join(LH_ROOT, 'stack-packs/packs'));
+  const stackPacksRoot = path.join(path.dirname(require.resolve('lighthouse-stack-packs')), 'packs');
```

Comment:
`path.join(a, b)` -> `${a}/${b}`. @brendankenny taught me that one. path delimiters are normalized by node so no issue for windows. this is given as input to glob so I _think_ it should still work ... if not i'll eat my words :)

Reply:
hahahha https://github.com/GoogleChrome/lighthouse/pull/10138#discussion_r371489799 wait nvm ignore me